subprojects/gtest.wrap

```diff
@@ -1,10 +1,10 @@
 [wrap-file]
-directory = googletest-release-1.8.0
```

Comment:
Wow, interesting that they removed the old link. Good change.

Reply:
That one was automatic. I ran `meson subprojects update`, and it pulled from the wrapdb. I figured you'd want the latest.

I also added `meson subprojects update` to the travis file, to see if that would fix pbcopper (in case the problem was the re-use of a directory on a re-used builder server), but it turned out that pbcopper was already up-to-date. So that line doesn't really need to be in travisci.yml.